### DISCUSSION TITLE
无法安装 termux-am

### DISCUSSION THREAD

**Discussion by alsjdnh** [termux/termux-app#3993] [Q&A] [answered]
![Screenshot_2024-06-05-23-46-10-996_com termux-edit](https://github.com/termux/termux-app/assets/131505167/6eecf569-5d4e-4cdb-9824-6da943861b17)
安装时报错
使用中国网络时同样

**Comment by agnostic-apollo** [comment, marked as answer]
Switch mirror with `termux-change-repo`, then run `apt update; apt install termux-tools termux-am`.
